proteus/WaveTools.pyx

```diff
@@ -19,17 +20,16 @@ import sys as sys
 
 
 def loadExistingFunction(funcName, validFunctions):
-    """ Checks if a function name is present in a list of known functions,
-    returns system exit if not present
+    """ Checks if a function name is present in a list of known functions, returns system exit if not present 
```

Comment:
@adimako - Thanks for putting together a PR to improve documentation!  Here are a few comments I have.

So I'm not sure how to handle this case.  

The standard calls for line length to be kept below 75 characters so that the comment can be read in the terminal.  Moreover, the standard states the top line comment of a function should be a single line.  If a longer description is required, it can be included in an extended comment following the top line comment.  A good example of this is shown: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html.

I'm not saying things need to be done this way, but I do find readability of the library really suffers in places because line lengths are allowed to grow so long so I'd personally like to see longer lines avoided where possible.